src/classes/TDTM_TriggerHandler.cls

```diff
@@ -242,4 +245,20 @@ public class TDTM_TriggerHandler {
                     
         return errors;
     }
+
+    /*******************************************************************************************************
+    * @description Modifies the global static collection of Trigger_Handler records to disable or enable
+    * a specific TriggerHander by SObjectName and ApexClass name.
+    * @param objName SObject Name
+    * @param className Apex Class Name
+    * @param state True (enable) | False (disable)
+    */
+    public static void toggleTriggerState(String objName, String className, Boolean state) {
```

Comment:
It's used in a few places where a specific trigger is being disabled entirely. It's a bit different than a pure recursion check, though that can certainly work for those that check the recursion state up front (not all do). The other advantage of completely disabling the trigger is that it prevents it the class from ever being instantiated at all, which should help with performance.